src/LinearSolverTypes_Multigrid.f90

```diff
@@ -551,11 +560,14 @@ SUBROUTINE setupPETScMG_LinearSolverType_Multigrid(solver,Params)
 
   !For now, only Galerkin coarse grid operators are supported.
   !  Galerkin means A_c = R*A*I
-  CALL PCMGSetGalerkin(solver%pc,PETSC_TRUE,iperr)
+!  CALL PCMGSetGalerkin(solver%pc,PETSC_TRUE,iperr)
+  !TODO: figure out Galerkin type
```

Comment:
there are several TODOs like this for advanced features

Reply:
Does this still work for older versions of PETSc, just not 3.12.4?

Reply:
I need to go through and add some version diffs for things like this

It compiles with 3.6.4, but I'm not sure it's right for that version.

What I'll do is go back through and make sure everything is consistent with 3.6.4...then we can go back and look at things for 3.12.4.

Reply:
👍

Reply:
alright, I think I've reinstated everything for 3.6.4.  Only some of the multigrid stuff and (maybe) parallel SuperLU features are not functional for 3.12.4.  Otherwise, should be good to go.